lib/base-compiler.js

```diff
@@ -94,6 +94,7 @@ Compile.prototype.runCompiler = function (compiler, options, inputFilename, exec
     }
 
     return this.exec(compiler, options, execOptions).then(function (result) {
+	result.inputFilename = inputFilename;
```

Comment:
Indentation problem. If you are using a Jetbrains IDE, applying the project's formatting should take care of most of my reviews